--- main_window.ui.old	2024-09-18 12:50:22.879130176 +0200
+++ main_window.ui	2024-09-18 12:50:31.663185646 +0200
@@ -285,8 +285,8 @@
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>239</red>
-              <green>239</green>
-              <blue>239</blue>
+              <green>41</green>
+              <blue>41</blue>
              </color>
             </brush>
            </colorrole>
